docker/Dockerfile

```diff
@@ -11,5 +11,9 @@ RUN mkdir -p $GOPATH/src/github.com/algorand
 WORKDIR $GOPATH/src/github.com/algorand
 RUN git clone https://github.com/algorand/go-algorand
 WORKDIR $GOPATH/src/github.com/algorand/go-algorand
+RUN apt-get -y install sudo
```

Comment:
Can you put the ENV command first, then combine the 3 apt-get commands into one?